int32 order
geometry_msgs/Point[] control_points
